docs/Table.md

```diff
@@ -149,6 +149,7 @@ This function accepts the following named parameters:
 |:---|:---|
 | className | Row-level class name |
 | columns | Array of React nodes |
+| key | Unique key within array of rendered rows |
```

Comment:
```suggestion
| key | Unique value within array of rendered rows |
```

Reply:
This is fine, but I chose the original wording to be consistent with the [documentation for List](https://github.com/bvaughn/react-virtualized/blob/master/docs/List.md#rowrenderer)